wifite/tools/airmon.py

```diff
@@ -58,9 +58,11 @@ class Airmon(Dependency):
     base_interface = None
     killed_network_manager = False
     use_ipiw = False
+    isdeprecated = False
 
     # Drivers that need to be manually put into monitor mode
     BAD_DRIVERS = ['rtl8821au']
+    DEPRECATED_DRIVERS = ['rtl8723cs']
```

Comment:
I named this variable as deprecated because "airmon-ng" has functions named as deprecated for drivers using "iwconfig" and with a flag variable "isdeprecated".